tests/integration/features/istio-jwt/istio_jwt.feature

```diff
@@ -87,4 +86,18 @@ Feature: Exposing endpoints with Istio JWT authorization strategy
     And JwtIssuerJwksNotMatch: There is an endpoint secured with JWT on path "/ip" with invalid issuer and jwks
     When JwtIssuerJwksNotMatch: The APIRule is applied
     And JwtIssuerJwksNotMatch: Calling the "/ip" endpoint with a valid "JWT" token should result in body containing "Jwks doesn't have key to match kid or alg from Jwt"
-    And JwtIssuerJwksNotMatch: Teardown httpbin service
\ No newline at end of file
+    And JwtIssuerJwksNotMatch: Teardown httpbin service
+
+  Scenario: Calling a httpbin endpoint secured with different JWT token from headers
+    Given JwtTokenFromHeaders: There is a httpbin service
+    When JwtTokenFromHeaders: The APIRule is applied
+    Then JwtTokenFromHeaders: Calling the "/headers" endpoint without a token should result in status between 400 and 403
+    And JwtTokenFromHeaders: Calling the "/headers" endpoint with a valid "JWT" token from header "x-jwt-token" and prefix "JwtToken" should result in status between 200 and 299
+    And JwtTokenFromHeaders: Teardown httpbin service
+
+  Scenario: Calling a httpbin endpoint secured with different JWT token from params
+    Given JwtTokenFromParams: There is a httpbin service
+    When JwtTokenFromParams: The APIRule is applied
+    And JwtTokenFromParams: Calling the "/ip" endpoint without a token should result in status between 400 and 403
```

Comment:
Could you add a negative check for token in default `authorization` header?

Reply:
Yes this makes sense, good catch. Incoming.